tensilelite/Tensile/Common.py

```diff
@@ -1121,7 +1121,7 @@ def getArchitectureName(gfxName):
     "HighPrecisionAccumulate":  False,            # f32 += f16*f16
     "SilentHighPrecisionAccumulate": False,       # Keep kernel names the same for HPA mode.  Useful for testing.
 
-    "SparseA":                  False,            # 4:2 Structured Sparse A Matrix
+    "SparseA":                  0,                # 4:2 Structured Sparse A Matrix, 0=Non Sparse, 1=Sparse Matrix A, 2=Sparse Matrix B
```

Comment:
Don't use SparseA as KernelProblemParameter since it is used for both A and B.
Revise it as "Sparse" at least

Reply:
done, please review again

Reply:
done, please review again